### PR TITLE
Flake8

### PR DESCRIPTION
**Pull request by schakrava**

### CODE REVIEW

**Comment by MFlyer** [comment]
Hi @schakrava are we going with flake8 tests on jenkins builds? that would be nice :)

M.

**Comment by schakrava** [comment]
yes @MFlyer . Please configure your IDE so all checks pass before opening pull requests!

**Comment by MFlyer** [comment]
Actually will have fun, working on a centralized git repo from office & home, had flake8 on server and will manually run it against files :wink:

File: src/rockstor/system/services.py [hunk 1]
```diff
@@ -239,20 +244,20 @@ def update_nginx(ip, port):
...
+                 re.search('listen.*80 default_server', lines[i+1]) is not
+                 None)):
+                # found legacy http server section. don't rewrite it.
                 http_server = True
-            if (not http_server and
-                re.search('listen.*default_server', lines[i]) is not None):
-                substr = 'listen %d default_server' % port
+            if ((not http_server and
+                 re.search('listen.*default_server', lines[i]) is not None)):
+                substr = 'listen %s:%d default_server' % (ip, port)
```
Comment (schakrava) [inline src/rockstor/system/services.py]: So this line switched with 256 causing the nginx/ui bug

File: src/rockstor/system/osi.py [hunk 2]
```diff
@@ -96,14 +96,14 @@ def run_command(cmd, shell=False, stdout=subprocess.PIPE,
...
+    except Exception as e:
         msg = ('Exception while running command(%s): %s' % (cmd, e.__str__()))
         raise Exception(msg)
 
     if (rc != 0):
         if (log):
-            e_msg = ('non-zero code(%d) returned by command: %s. output: %s error:'
-                     ' %s' % (rc, cmd, out, err))
+            e_msg = ('non-zero code({0}) returned by command: {1}. output: '
+                     '{2} error: {3}'.format(rc, cmd, out, err))
```
Comment (phillxnet) [inline src/rockstor/system/osi.py]: @schakrava Is this an intended edit, ie from %d %s etc to {0}  {1}, don't we loose formatting capability this way?
Due to merge conflicts I'm hand converting my pr so need to know what you intended here.

**Pull request merged**